tests/e2e/autoscaling/autoscaler.go

```diff
@@ -404,7 +404,7 @@ func createPodSpec(requestCPU int64) (result v1.PodSpec) {
 		Containers: []v1.Container{
 			{
 				Name:  "container",
-				Image: "nginx:1.15",
+				Image: "k8s.gcr.io/e2e-test-images/agnhost:2.36",
```

Comment:
I would like to know what's the diff between these two images? thanks.

Reply:
nginx image doesn't support Windows.